packages/manager/modules/netapp/src/dashboard/volumes/dashboard/snapshots/routing.js

```diff
@@ -86,6 +87,26 @@ export default /* @ngInject */ ($stateProvider) => {
               )
               .then((snapshots) => snapshots.reduce((a, b) => a + b, 0)),
           ),
+
+      snapshotPolicies: /* @ngInject */ (getSnapshotPolicies) =>
+        getSnapshotPolicies().catch(() => []),
+      currentPolicy: /* @ngInject */ ($http, serviceName, volumeId) =>
+        $http
+          .get(
+            `/storage/netapp/${serviceName}/share/${volumeId}/snapshotPolicy`,
+          )
+          .then(({ data }) => data)
+          .catch(() => ({})),
```

Comment:
I think it's better to have HTTP request into a service file.

Reply:
Following call is only used here.